discord/member.py

```diff
@@ -540,7 +541,7 @@ def display_name(self) -> str:
         if they have a guild specific nickname then that
         is returned instead.
         """
-        return self.nick or self.name
+        return self.nick or self.global_name if self._user.is_migrated else self.name or self.name
```

Comment:
@Om1609 does that even makes sense 

![cry](https://media.tenor.com/bMSZQ4j3CQkAAAAC/anime-cry.gif)

Reply:
Uhh wtf?

Reply:
i dunno how to do it better

https://github.com/Aiko-IT-Systems/DisCatSharp/blob/a1bd54efb9d09d069257d6f954cab6ba3bedbc28/DisCatSharp/Entities/Guild/DiscordMember.cs#L152-L159